index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="navbar">
+        <img class="pic1" src="image_1.png" alt="geometry">
+        <img class="menu" src="ic_menu.png" alt="Menu">
+    </div>
+
+    <div class="headline">
+        <h1 class="name" >Emilia Zambrano</h1>
```

Comment:
- [ ] Kindly use `section` tag instead of `div` here `<div class="headline">`, it's more semantic.

Reply:
resolved 👍